src/core/Form/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { axeTest } from '../../../utils/test/axe';
+import { render } from '@testing-library/react';
+import { Checkbox } from './Checkbox';
+
+const createTestCheckbox = (
+  large: boolean,
+  error: boolean,
+  defaultChecked: boolean,
+  labelText: string,
+  hint?: string,
+  statusText?: string,
+  dataTestId?: string,
+) => (
+  <Checkbox
+    variant={large ? 'large' : 'small'}
+    defaultChecked={defaultChecked}
+    status={error ? 'error' : 'default'}
+    hintText={hint}
+    statusText={statusText}
+    data-testid={dataTestId}
+    id="test"
+  >
+    {labelText}
+  </Checkbox>
+);
+
+const RegularTestCheckbox = createTestCheckbox(
```

Comment:
Looks good, maybe add disabled as well since that might be quite commonly used and depends on styles.